### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExtensions_Screenreader:NVDA does not read the content written under treeview.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51968]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
ScreenReader,NVDA[2018.2],JAWS[18.0.5038],MAS1.3.1
**Repro Steps:**
1. Launch VS Code.
2.Navigate to View(Alt+V) menu option and select "Extensions" from dropdown.
3.Navigate to tree view "Installed" and "Recommendation" under the search extension.
4. Check what NVDA is reading.

**Actual:**
NVDA does not read the content written under the treeview.
Same issue observed in Jaws also.
**Expected:**

NVDA should read the  content which is present on the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**

User will not able to know which content is written on the screen.
**MAS Reference:**
[https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference:**

![accessibility a11y_vscode_viewextensions_screenreader1](https://user-images.githubusercontent.com/39204909/41455645-796041a8-709b-11e8-916a-20c3cac456c4.png)
![accessibility a11y_vscode_viewextensions_screenreader](https://user-images.githubusercontent.com/39204909/41455686-9920e5a6-709b-11e8-8714-742d9ec6db36.png)
